### PR TITLE
add monitoring approvers for cherry-pick-approved

### PR DESCRIPTION
**Pull request by juzhao**
Based on doc [Designating QE Z-Stream Cherry-Pick Approvers for a Repository](https://docs.google.com/document/d/1PC87sSFa_zGCk95kXDW-wrVxnlgBmkHqpOgQnd4bbUw/edit#heading=h.m4cd5pju584k), adds @juzhao as a cherry pick approver and @wzheng1 as secondary cherry pick approvers. 

@wzheng1 @lihongyan1 Please help to review in case I missed other changes

### CODE REVIEW

**Comment by juzhao** [comment]
@openshift/openshift-team-monitoring please help to review

File: core-services/prow/02_config/openshift/cluster-monitoring-operator/_pluginconfig.yaml [hunk 1]
```diff
@@ -10,6 +10,10 @@ label:
       - arajkumar
       - simonpasquier
       label: backport-risk-assessed
+      allowed_users:
```
Comment (simonpasquier) [inline core-services/prow/02_config/openshift/cluster-monitoring-operator/_pluginconfig.yaml:13]: shouldn't it be a new item under the openshift/cluster-monitoring-operator key?

```suggestion
    - allowed_users:
```

Comment (simonpasquier) [reply core-services/prow/02_config/openshift/cluster-monitoring-operator/_pluginconfig.yaml:13]: like in https://github.com/openshift/release/blob/6f5b16a432d9628939f7a96e3efe32e4d0a66e69/core-services/prow/02_config/openshift/machine-config-operator/_pluginconfig.yaml#L7-L16

File: core-services/prow/02_config/openshift/cluster-monitoring-operator/_pluginconfig.yaml [hunk 2]
```diff
@@ -3,13 +3,17 @@ label:
     openshift/cluster-monitoring-operator:
     - allowed_teams:
       - openshift-patch-managers
-      allowed_users:
+    - allowed_users:
```
Comment (simonpasquier) [inline core-services/prow/02_config/openshift/cluster-monitoring-operator/_pluginconfig.yaml:6]: thanks for fixing it, my bad!

**Review by simonpasquier — COMMENTED** [review]
you need to run `make prow-config`

**Pull request merged**